File: app/src/main/java/com/alumnus/zebra/db/entity/CsvFileLogEntity.java
package com.alumnus.zebra.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.Index;
import androidx.room.Insert;
import androidx.room.PrimaryKey;

@Entity(tableName = "csv_file_log", indices = {@Index(value = {"file_name"}, unique = true)})
public class CsvFileLogEntity {

    @PrimaryKey(autoGenerate = true)
    public long id;

    @ColumnInfo(name = "file_name")
    public String file_name;

    @ColumnInfo(name = "count")
    public long count;

    public CsvFileLogEntity(String file_name, long count) {
        this.file_name = file_name;
        this.count = count;
    }
}

File: app/src/main/java/com/alumnus/zebra/db/entity/CsvFileLogEntity.kt
package com.alumnus.zebra.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Index
import androidx.room.PrimaryKey

@Entity(tableName = "csv_file_log", indices = [Index(value = ["file_name"], unique = true)])
data class CsvFileLogEntity(

        @ColumnInfo(name = "file_name")
        var file_name: String,

        @ColumnInfo(name = "count")
        var count: Long,

        @PrimaryKey(autoGenerate = true)
        @ColumnInfo(name = "id")
        var id: Long = 0
)